tap-schema/build-all

```diff
@@ -4,9 +4,17 @@
 # values that control the presentation order of schemas.  This is respected by the Portal Aspect.
 #
 ./build mock
-./build idfprod ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
-./build idfint ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
-./build idfdev ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
-./build idfsso ../yml/dp03.yaml
-./build usdf-dev ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/oga_live_obscore.yaml
-./build usdf-prod ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml
+
+./build tap-schema-idfprod-tap ../yml/dp02_dc2.yaml ../yml/dp01_dc2.yaml ../yml/dp02_obscore.yaml
+./build tap-schema-idfint-tap ../yml/dp02_dc2.yaml ../yml/dp01_dc2.yaml ../yml/dp02_obscore.yaml
+./build tap-schema-idfdev-tap ../yml/dp02_dc2.yaml ../yml/dp01_dc2.yaml ../yml/dp02_obscore.yaml
+./build tap-schema-usdf-dev-tap ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml
+./build tap-schema-usdf-prod-tap ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml
+
+./build tap-schema-usdf-dev-obstap ../yml/dp02_obscore.yaml ../yml/oga_live_obscore.yaml
```

Comment:
This is not correct - "live ObsCore" is on a different back-end (Postgres) DB from DP0.2 ObsCore, which is on Qserv.

Let's call this one "tap-schema-usdf-dev-live-obscore", and it has only `../yml/oga_live_obscore.yaml` on it.

Reply:
I'm trying to name it after the environment + tap service they are using just to be explicit.